src/GraphQL.SystemTextJson/JsonConverterBigInteger.cs

```diff
@@ -16,13 +16,22 @@ public sealed class JsonConverterBigInteger : JsonConverter<BigInteger>
         public override BigInteger Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
             => TryGetBigInteger(ref reader, out var bi) ? bi : throw new JsonException();
 
+#if NETSTANDARD2_0
+        public static bool TryGetBigInteger(ref Utf8JsonReader reader, out BigInteger bi)
+        {
+            var byteArray = reader.HasValueSequence ? reader.ValueSequence.ToArray() : reader.ValueSpan.ToArray();
+            var str = Encoding.UTF8.GetString(byteArray);
+            return BigInteger.TryParse(str, out bi);
```

Comment:
Unfortunately, in the case of netstandard, we have to use methods that allocate memory.